corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -1007,6 +1008,42 @@ hqDefine('geospatial/js/models', [
             utils.downloadCsv(casesToExport, headers, cols, 'Case Assignment Export');
         };
 
+        self.acceptAssignments = function () {
+            let caseIdToOwnerId = {};
+            for (const caseItem of self.mapModel.caseMapItems()) {
+                const caseData = self.mapModel.caseGroupsIndex[caseItem.itemId];
+                if ('assignedUserId' in caseData && caseData.assignedUserId) {
+                    caseIdToOwnerId[caseData.item.itemId] = caseData.assignedUserId;
+                }
+            }
+            let requestData = {
+                'case_id_to_owner_id': caseIdToOwnerId,
+                'include_related_cases': self.includeRelatedCases(),
+            };
+
+            $.ajax({
+                type: 'post',
+                url: reassignCasesUrl,
+                dataType: 'json',
+                data: JSON.stringify(requestData),
+                contentType: "application/json; charset=utf-8",
+                success: function (response) {
+                    if (!response.success) {
+                        return alertUser.alert_user(response.message, 'danger');
+                    }
+                    alertUser.alert_user(response.message, 'success', false, true);
+                },
+                error: function (response) {
+                    const responseText = response.responseText;
+                    if (responseText) {
+                        alertUser.alert_user(responseText, 'danger');
+                    } else {
+                        alertUser.alert_user(gettext(unexpectedErrorMessage), 'danger');
```

Comment:
nit: I have never seen a variable passed to gettext but always a string. I assume this should also work as expected.

Reply:
Can confirm that this works